website/docs/intro/install/index.mdx

```diff
@@ -43,9 +43,9 @@ You can install OpenTofu via a wide range of methods. Please select your operati
         {
             type: "link",
             href: "/docs/intro/install/macos",
```

Comment:
I believe this should be changed too. You can test the website using the new container setup in the [website repo](https://github.com/opentofu/opentofu.org) to make sure everything works.